script/protoc-swagger-gen.sh

```diff
@@ -21,7 +21,7 @@ done
 # combine swagger files
 # uses nodejs package `swagger-combine`.
 # all the individual swagger files need to be configured in `config.json` for merging
-swagger-combine ./client/docs/config.json \
+.cache/node_modules/.bin/swagger-combine ./client/docs/config.json \
```

Comment:
it should pick swagger-combine from PATH as we manually set it. doesn't it work?

Reply:
simply running `make codegen` wasn't working locally for me without this change.